src/CUDA/cudaError.h

```diff
@@ -0,0 +1,39 @@
+//////////////////////////////////////////////////////////////////////////////////////
+//// This file is distributed under the University of Illinois/NCSA Open Source License.
+//// See LICENSE file in top directory for details.
+////
+//// Copyright (c) 2019 QMCPACK developers.
+////
+//// File developed by: Ye Luo, yeluo@anl.gov, Argonne National Laboratory
+////
+//// File created by: Ye Luo, yeluo@anl.gov, Argonne National Laboratory
+////////////////////////////////////////////////////////////////////////////////////////
+
+
+#ifndef QMCPLUSPLUS_CUDA_ERROR_H
+#define QMCPLUSPLUS_CUDA_ERROR_H
+
+#include <iostream>
+#include <string>
+#include <sstream>
+#include <stdexcept>
+#include <cuda_runtime_api.h>
+
+#define cudaErrorCheck(ans, cause) { cudaAssert((ans), cause, __FILE__, __LINE__); }
+/// prints CUDA error messages. Always use cudaErrorCheck macro.
+inline void cudaAssert(cudaError_t code, const std::string& cause, const char *filename, int line, bool abort=true)
```

Comment:
The abort parameter appears not to have any effect.

Reply:
Fixed.